ts/src/header-validator/validate-source.ts

```diff
@@ -237,6 +238,42 @@ function aggregationKeys(j: Json, ctx: Context): Maybe<AggregationKeys> {
   )
 }
 
+export function aggregatableBucketLength(
+  s: string,
+  ctx: Context,
+  errPrefix: string = ''
```

Comment:
Since `aggregatableBucketLength` is only called in one location, let's get rid of the `errPrefix` parameter and just inline the `'bucket '` prefix.